src/libs/actions/Report.js

```diff
@@ -1232,6 +1243,9 @@ function deleteReportComment(reportID, reportAction) {
         ],
     };
 
+    // Clear the draft comment for action if any.
+    saveReportActionDraft(reportID, reportAction.reportActionID, '');
+
```

Comment:
To answer your question from the issue, showdeleteModal does not take any callback for now so we can't hook this call to the deleteModal. But I think it is fairly safe to call this method here. But let me know if you think this is confusing.